telegram/_message.py

```diff
@@ -3570,6 +3601,9 @@ def text_markdown(self) -> str:
 
             * |custom_emoji_formatting_note|
 
+        .. deprecated:: NEXT.VERSION
+            |custom_emoji_md1_deprecation|
```

Comment:
this tells me the whole method is deprecated, I think we should just modify `custom_emoji_formatting_note` for this or put this in a Caution block instead.

Reply:
![image](https://user-images.githubusercontent.com/22366557/236620477-0f2fc8a6-a32c-4d47-9c73-fe051bcadea8.png)

I'd read that as "the following is behavior is deprecated since NEXT.VERSION: …" …